packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1696,6 +1696,60 @@ abstract class ParentDataWidget<T extends ParentData> extends ProxyWidget {
 /// for that inherited widget using [BuildContext.dependOnInheritedWidgetOfExactType]
```

Comment:
for clarity let's put a header like "## The \`.of\` method convention" or "Implementing \`.of\` methods" above the paragraph that starts "The convention...", and then a similar header above your new text e.g. "## Calling `\.of\` methods" or some such.

Reply:
done